core/data_migration_scripts/4_10_0_stages/EE_DMS_4_10_0_Confirm_Email.dmsstage.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ *
+ * EE_DMS_4_10_0_Confirm_Email
+ *
+ * @package         Event Espresso
+ * @subpackage
+ * @author              Brent Christensen
+ *
+ */
+class EE_DMS_4_10_0_Confirm_Email extends EE_Data_Migration_Script_Stage_Table
```

Comment:
this entire file can actually be deleted...
we don't need it.

A Data Migration Script (DMS) is needed when we need to either:

- move existing data to a new table and/or field 

- modify existing data in some manner

In this PR, the new email confirmation question gets added as part of the database initialization process that you added in `EEH_Activation::initialize_system_questions()`.
I've checked out this branch and tested locally and the question data gets successfully added to the database without this DMS even running...

...you're probably wondering right now what I mean by "without this DMS even running" right?

In order for a DMS script to actually run during an update a couple of things have to happen:

1. the version number for the application has to be incremented, ex: 4.10.4 => 4.10.5

2. the DMS class has to be instantiated and added to the `_migration_stages` array for the `EE_Data_Migration_Script_Base` class of the current version, which in this case would be in the constructor of `core/data_migration_scripts/EE_DMS_Core_4_10_0.dms.php`.

an example of the latter would be:

```php
        $this->_migration_stages = array(
            new EE_DMS_4_10_0_Event_Question_Group(),
            new EE_DMS_4_10_0_Confirm_Email(),
        );
```

but again, we don't even need that because the new question is already added to the db upon activation.